spec/tapioca/cli/dsl_spec.rb

```diff
@@ -804,8 +804,6 @@ class Post
           OUT
 
           assert_equal(<<~ERR, result.err)
-
-            Warning: No constants found in: path/to/nowhere.rb
```

Comment:
I intentionally remove this warning since we already show an error if paths are given but no constants are found.